src/java/org/apache/cassandra/db/filter/RowFilter.java

```diff
@@ -1081,6 +1106,112 @@ protected Kind kind()
         }
     }
 
+
+    // VSTODO make generic?
```

Comment:
Not sure what kind of generification you're thinking of here -- Expression isn't generic, and making the type of vector that Geo operates on generic doesn't make sense to me

Reply:
Generic as in supporting multiple kinds of bounded ann. We can do that later since it's an internal class. I'll remove the comment.